rpcs3/Emu/RSX/VK/VKRenderPass.h

```diff
@@ -10,4 +10,11 @@ namespace vk
 	VkRenderPass get_renderpass(VkDevice dev, u64 renderpass_key);
 
 	void clear_renderpass_cache(VkDevice dev);
+
+	// Renderpass scope management helpers.
+	// NOTE: These are not thread safe by design.
+	void begin_renderpass(VkDevice dev, VkCommandBuffer cmd, u64 renderpass_key, VkFramebuffer target, const coordu& framebuffer_region);
+	void begin_renderpass(VkCommandBuffer cmd, VkRenderPass pass, VkFramebuffer target, const coordu& framebuffer_region);
+	void end_renderpass(VkCommandBuffer cmd);
+	bool is_renderpass_open(VkCommandBuffer cmd);

```

Comment:
```suggestion
	bool is_renderpass_open(VkCommandBuffer cmd) const;

```

Reply:
It's not owned by a class, it's a global function. Static would make copies for every file that includes and inline is pretty much guaranteed by compiler for this. I could add inline hint if you like, but it's more annotation than functional change.

Reply:
oh, yeah, i didn't see that it's not a member function by looking through it on my phone 👍